html_to_playlist.py

```diff
@@ -1,63 +1,82 @@
 import argparse
-
 import pandas as pd
 
+# local
 import utils
+from spotify_abs_cls import SpotifuHandler
 
+# constants
 IS_PUBLIC_PLAYLIST = True
 
-
 def parse_args():
     parser = argparse.ArgumentParser(fromfile_prefix_chars='@')
-    parser.add_argument('--html-file', '-f', type=str, help='HTML file path',
-                        default='bidud_party.html', )
-    parser.add_argument('--spotify-yaml', '-s', help='Spotify credentials file',
+    parser.add_argument('--spotify-yaml-path', '-s', help='Spotify credentials file',
```

Comment:
I think I used underscores (spotify_yaml_path) and you use dashes (spotify-yaml-path).
Maybe we should choose one standard and conform?

Reply:
I actually noticed that. I generally like dash-separated, since they are converted automatically to underscore-separated. As a bonus, you can usually used just the first word (use just spotify rather than spotify-yaml-path) when using dashes. What do you use at Razor? I'm fine with both (we use dashes).